app/Http/Controllers/AuthenticationController.php

```diff
@@ -7,16 +7,39 @@
 use Lcobucci\JWT\Parser;
 
 use App\User;
+use App\Categories;
+use App\Challenges;
+use App\Content;
+
 
 class AuthenticationController extends Controller
 {
-    public function login(Request $request) {
+    public function login(Request $request)
+    {
         $user = User::where('email', $request->email)->first();
-
+        $categories = Categories::all();
+        $fullContent = [];
+        foreach ($categories as $category) {
+            $unique_chal = Challenges::where('category_id', $category->id)->get();
+            // array_push($challenges,$unique_chal);
```

Comment:
May want to remove these commented out items in this file, unless you're keeping them there for future iteration/work. I noticed you have some placeholder commented items in some other places, so if that's the case, you can keep it for now.